portal/app/api/v1/endpoint/sonnen_data.py

```diff
@@ -0,0 +1,41 @@
+from app.api.v1 import CsrfExemptAuth
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from app.models import FarmDevice, Home, Device
+from rest_framework import (viewsets, serializers, status)
+from rest_framework.authentication import TokenAuthentication
+import requests
```

Comment:
A lot of imports here that aren't being used - remove them before merging this in.